protos/sigstore_bundle.proto

```diff
@@ -55,15 +55,15 @@ message VerificationMaterial {
         }
         // This is the inclusion promise and/or proof, where
         // the timestamp is coming from the transparency log.
-        repeated dev.sigstore.rekor.v1.TransparencyLogEntry tlog_entries = 3;
+        repeated dev.sigstore.rekor.v1.TransparencyLogEntry tlog_entries = 3 [(google.api.field_behavior) = REQUIRED];
```

Comment:
There’s no guarantee an entry is added to a tlog.

Reply:
Ah, that is interesting. Technically speaking, one can query the log to check the presence, however, it would be nice if the inclusion proof was attached in the first place.

I wonder if it makes sense to add sigstore-specific annotations like 'required for offline bundle'

WDYT?

Reply:
There’s still the case where no tlog is used if you’re using only a signed time stamp (in cases of private Sigstore instances). I think we would end up needing multiple annotations, for example how to verity with a timestamp, how to verify with a tlog, how to verify with both. That may be better to have in comments or a part of a reference implementation of verification logic.